modules/javafx.controls/src/main/java/javafx/scene/control/Tooltip.java

```diff
@@ -1074,6 +1092,9 @@ private void install(Node node, Tooltip t) {
             node.addEventHandler(MouseEvent.MOUSE_MOVED, MOVE_HANDLER);
             node.addEventHandler(MouseEvent.MOUSE_EXITED, LEAVING_HANDLER);
             node.addEventHandler(MouseEvent.MOUSE_PRESSED, KILL_HANDLER);
+            t.addEventFilter( MouseEvent.MOUSE_ENTERED_TARGET, TOOLTIP_MOUSE_ENTERED );
```

Comment:
Code style: remove spaces after `(` and before `)` here and similar method calls elsewhere.